views.py

```diff
@@ -0,0 +1,21 @@
+from flask import Blueprint, jsonify
+from sqlalchemy import desc
+
+
+def config_routes(db):
```

Comment:
Will this still work since we moved db?

Reply:
Why do you embed the view in another view?

Reply:
I wrapped the route (or the view) in config_routes function to be able to call it (and all future routes) for normal app and for the testing app, db is an argument represents normal db or db for testing depending on the case. About the question about if the app is still working, the answer is yes, I just ran the app and it is working, also pytest is green :).

Reply:
Looks a bit strange to me but let's go with that for now